packages/roosterjs-content-model-core/lib/command/paste/generatePasteOptionFromPlugins.ts

```diff
@@ -25,6 +25,7 @@ export function generatePasteOptionFromPlugins(
         processorOverride: {},
         styleSanitizers: {},
         attributeSanitizers: {},
+        processNonVisibleElements: false,
```

Comment:
Do we need to get this value from editor? You can get the settings from `editor.getEnvironment().domToModelSettings`